tools/rcverify.sh

```diff
@@ -114,6 +114,7 @@ EXPECTED=$(cat "$DIR/$TGZ.sha512")
 CMD="cd $DIR && gpg --print-md SHA512 '$TGZ'"
 SHA=$(eval $CMD)
 validate "$EXPECTED" "$SHA" "$CMD"
+echo "SHA512: $(tput setaf 6)$SHA$(tput sgr0)"
```

Comment:
You can emit this on the line before:
```
validate "$EXPECTED" "$SHA" "$CMD" "SHA512: $(tput setaf 6)$SHA$(tput sgr0)"
```

when "this" matches "that", the fourth argument is printed so the output will look like this:
```
computing sha512 and validating... passed (SHA512: openwhisk-runtime-go-1.13.0-incubating-sources.tar.gz: 
E9B111CC 28B61FD2 A22FDE7C F838F063 01C4DD79 E568169D 62667C4D D8D5260C 23446127
 406000E5 930DB6E8 9D4D50ED 503B1654 9AC92F17 93BCC03F E5FB5D2F)
```